config/config_test.go

```diff
@@ -129,3 +129,23 @@ func TestParse(t *testing.T) {
 		}
 	}
 }
+
+func TestParseFields(t *testing.T) {
```

Comment:
**question:** Should this be more specific to ReportGroups?

Reply:
I was thinking that this would eventually cover testing that all fields are parsed correctly, but I also wanted to scope my changes to a minimum. I added a comment to hopefully make that clearer. What do you think?